docs/dev/connectors/elasticsearch2.md

```diff
@@ -1,173 +0,0 @@
----
```

Comment:
Can you replace the es2 page with a redirect to the new page?
This way existing links are not broken.